api/v1alpha7/types.go

```diff
@@ -163,6 +163,15 @@ type RootVolume struct {
 	AvailabilityZone string `json:"availabilityZone,omitempty"`
 }
 
+type DataVolume struct {
+	NameSuffix       string `json:"nameSuffix"`
```

Comment:
`omitempty`? If not, it's probably also `+required`.

Reply:
As it is now, it comes out in the `required` keys in the CRD schema. Not sure it can be `omitempty` - if you want a data volume you have to name it I think